### PR TITLE
User management is completed with simple basic UI

### PR DESCRIPTION
**Pull request by AnvarjonShavqiyev**
Day5 tasks is completed.
You can check user creation with Auth page, user editing is via actions column of table.

### CODE REVIEW

**Comment by VladKetris** [comment]
please add UI screenshots

File: my-react-app/src/components/navbar/Navbar.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.navbar__wrapper{
+    background-color: #000;
+    color: #fff;
+    padding: 15px 0;
+}
+.navbar{
```
Comment (VladKetris) [inline my-react-app/src/components/navbar/Navbar.scss:6]: please add line break before next selector

File: my-react-app/src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { useDispatch } from 'react-redux'
+import { Container } from '../../utils/Utils'
+import './Navbar.scss'
+import { AppDispatch } from '../../redux/store/Store'
+import { logOut } from '../../redux/features/AuthSlice'
+const Navbar = () => {
+  const dispatch = useDispatch<AppDispatch>()
+  const doLogOut = () => {
```
Comment (VladKetris) [inline my-react-app/src/components/navbar/Navbar.tsx:8]: handler functions naming usually starts with 'on': "onLogOut", "onModalOpen", etc.

File: my-react-app/src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { useDispatch } from 'react-redux'
```
Comment (VladKetris) [inline my-react-app/src/components/navbar/Navbar.tsx:1]: where are all the semicolons?

File: my-react-app/src/components/navbar/Navbar.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const Navbar = () => {
+  const dispatch = useDispatch<AppDispatch>()
+  const doLogOut = () => {
+    dispatch(logOut())
+  }
+  return (
+    <div className='navbar__wrapper'>
+        <Container>
+          <div className='navbar'>
+            <h2>User managemant</h2>
```
Comment (VladKetris) [inline my-react-app/src/components/navbar/Navbar.tsx:15]: typo 'management'

File: my-react-app/src/components/users-table/UserTable.tsx [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+interface EditableCellProps extends React.HTMLAttributes<HTMLElement> {
+  editing: boolean;
+  dataIndex: string;
+  title: any;
+  inputType: 'number' | 'text';
+  record: User;
+  index: number;
+}
+
+const EditableCell: React.FC<React.PropsWithChildren<EditableCellProps>> = ({
```
Comment (VladKetris) [inline my-react-app/src/components/users-table/UserTable.tsx:26]: please create separate component

File: my-react-app/src/pages/Auth/Auth.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
+.auth-wrapper {
```
Comment (VladKetris) [inline my-react-app/src/pages/Auth/Auth.scss:1]: why don't you using nesting since it's scss ?

File: my-react-app/src/pages/Auth/Auth.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import { Link, useParams } from "react-router-dom";
+import "./Auth.scss";
+import SignIn from "./signin/Signin";
+import SignUp from "./signup/Signup";
+import React from "react";
```
Comment (VladKetris) [inline my-react-app/src/pages/Auth/Auth.tsx:5]: please add line break after imports

File: my-react-app/src/pages/Auth/signin/Signin.tsx [hunk 8]
```diff
@@ -0,0 +1,48 @@
+import { Form, Input, Button } from 'antd';
+import { MailOutlined, LockOutlined } from '@ant-design/icons';
+import { useDispatch } from 'react-redux';
+import { AppDispatch } from '../../../redux/store/Store';
```
Comment (VladKetris) [inline my-react-app/src/pages/Auth/signin/Signin.tsx:4]: please read about import aliases
https://blog.logrocket.com/using-path-aliases-cleaner-react-typescript-imports/

File: my-react-app/src/pages/Auth/signin/Signin.tsx [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+import { User } from '../../../types';
+
+const Signin = () => {
+  const dispatch = useDispatch<AppDispatch>()
+  const onFinish = (values:any) => {
+    dispatch(signInUser({ email: values.email }));
+  };
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', marginTop:'40px', width:"100%"}}>
```
Comment (VladKetris) [inline my-react-app/src/pages/Auth/signin/Signin.tsx:15]: use classname instead inline styles please

File: my-react-app/src/pages/Auth/signin/Signin.tsx [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+          rules={[{ required: true, message: 'Please input your Password!' }]}
+        >
+          <Input
+            prefix={<LockOutlined className="site-form-item-icon" />}
+            type="password"
+            placeholder="Password"
+          />
+        </Form.Item>
+        <Form.Item>
+          <Button type="primary" htmlType="submit" className="signin-form-button" style={{ width: '100%' }}>
```
Comment (VladKetris) [inline my-react-app/src/pages/Auth/signin/Signin.tsx:39]: please read about BEM class naming convention
https://en.bem.info/methodology/naming-convention/

File: my-react-app/src/pages/Auth/signup/Signup.tsx [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+            placeholder="Password"
+          />
+        </Form.Item>
+        <Form.Item
+          name="retypePassword"
+          dependencies={['password']}
+          hasFeedback
+          rules={[
+            { required: true, message: 'Please retype your Password!' },
+            ({ getFieldValue }) => ({
```
Comment (VladKetris) [inline my-react-app/src/pages/Auth/signup/Signup.tsx:54]: please move this function outside jsx

File: my-react-app/src/components/navbar/Navbar.scss (hunk 1 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/components/navbar/Navbar.scss:6]: I got it

File: my-react-app/src/components/navbar/Navbar.tsx (hunk 2 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/components/navbar/Navbar.tsx]: I understood.Fixed

File: my-react-app/src/components/navbar/Navbar.tsx (hunk 4 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/components/navbar/Navbar.tsx]: fixed

File: my-react-app/src/pages/Auth/Auth.tsx (hunk 7 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Auth/Auth.tsx:5]: I understood.

File: my-react-app/src/pages/Auth/signin/Signin.tsx (hunk 8 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Auth/signin/Signin.tsx:4]: I understood my mistake, after read it.Thanks

File: my-react-app/src/pages/Auth/signin/Signin.tsx (hunk 9 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Auth/signin/Signin.tsx:15]: Yeah, i got it

File: my-react-app/src/pages/Auth/signin/Signin.tsx (hunk 10 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Auth/signin/Signin.tsx:39]: I understood BEM

File: my-react-app/src/pages/Auth/signup/Signup.tsx (hunk 11 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Auth/signup/Signup.tsx:54]: Okey, i got it

File: my-react-app/src/components/users-table/UserTable.tsx (hunk 5 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/components/users-table/UserTable.tsx]: Seperated

File: my-react-app/src/pages/Auth/signup/Signup.tsx (hunk 11 above)
Comment (AnvarjonShavqiyev) [reply my-react-app/src/pages/Auth/signup/Signup.tsx]: Fixed

**Pull request merged**